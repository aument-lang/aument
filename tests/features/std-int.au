print int(2) + 1;
print int("3") + 1;
print int("4ab") + 1;