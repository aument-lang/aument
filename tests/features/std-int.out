int;3
int;4
int;5